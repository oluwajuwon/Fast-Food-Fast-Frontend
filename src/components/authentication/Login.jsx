import React from 'react';

const Login = () => (
  <div>
  This is login
    <p className="Class-inro">Hello my people</p>
  </div>
);

export default Login;
