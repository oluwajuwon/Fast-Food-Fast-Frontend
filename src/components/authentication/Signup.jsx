import React from 'react';

const Signup = () => (
  <div>
  This is Signup
    <p>Hello everyone</p>
  </div>
);

export default Signup;
