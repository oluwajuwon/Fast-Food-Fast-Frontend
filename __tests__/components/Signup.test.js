import React from 'react';
import { shallow } from 'enzyme';
import { Signup } from '../../src/components/authentication/Signup';

const props = {
  message: '',
  isSuccessful: '',
};
describe('Test for login component', () => {
  it('Should contain', () => {
    const wrapper = shallow(<Signup {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
