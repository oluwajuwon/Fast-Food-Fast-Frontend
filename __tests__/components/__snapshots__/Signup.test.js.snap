// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for login component Should contain 1`] = `
<div
  className="box-section"
>
  <NavBar />
  <section>
    <div
      className="box-container"
    >
      <div
        className="innerbox-container"
      >
        <div
          className="form"
        >
          <h2>
            Create an account
          </h2>
          <form
            id="user-Signup"
            onSubmit={[Function]}
          >
            <div>
              <p
                className="red-text text-center"
              />
              <p
                className=""
              >
                Full name
              </p>
              <input
                className="textbox"
                id="full-name"
                onChange={[Function]}
                placeholder="Enter full name here..."
                required={true}
                type="text"
              />
            </div>
            <div>
              <p
                className=""
              >
                Username
              </p>
              <input
                autoComplete="false"
                className="textbox"
                id="username"
                onChange={[Function]}
                placeholder="Enter username here..."
                required={true}
                type="text"
              />
            </div>
            <div>
              <p
                className=""
              >
                Email
              </p>
              <input
                className="textbox"
                id="email"
                onChange={[Function]}
                placeholder="Enter email here..."
                required={true}
                type="email"
              />
            </div>
            <div>
              <p
                className=""
              >
                Password
              </p>
              <input
                className="textbox"
                id="password"
                onChange={[Function]}
                onKeyUp={[Function]}
                placeholder="Enter password here..."
                required={true}
                type="password"
              />
            </div>
            <div>
              <p
                className=""
              >
                Confirm Password
              </p>
              <input
                className="textbox"
                id="password-match"
                onChange={[Function]}
                onKeyUp={[Function]}
                placeholder="Enter password again..."
                required={true}
                type="password"
              />
            </div>
            <br />
            <div
              className=""
            >
              <button
                className="btn blue-bg-colour white-text"
                id="submit-signup"
                type="submit"
              >
                Sign up
              </button>
              <Link
                className="secondary-text-color"
                replace={false}
                to="/login"
              >
                Already have an account? Login
              </Link>
            </div>
          </form>
        </div>
      </div>
    </div>
  </section>
  <Footer />
</div>
`;
